support/k8s/k8s-workload-registrar/mode-crd/api/spiffeid/v1beta1/spiffeid_webhook.go

```diff
@@ -116,5 +118,11 @@ func (s *SpiffeID) validateSpiffeID() error {
 		}
 	}
 
+	for _, dnsName := range s.Spec.DnsNames {
+		if err := x509util.ValidateDNS(dnsName); err != nil {
+			return fmt.Errorf("invalid DNS name: %v", err)
```

Comment:
the errors returned by `ValidateDNS` don't repeat the DNS name passed for validation. It may be worth adding the DNS name to the error message so that there is some helpful context about which DNS name was invalid:
```suggestion
			return fmt.Errorf("invalid DNS name %q: %v", dnsName, err)
```